spot-ml/src/main/scala/org/apache/spot/spotldacwrapper/SpotLDACSchema.scala

```diff
@@ -8,12 +8,16 @@ object SpotLDACSchema {
   // modelDF columns
   val DocumentName = "document_name"
   val DocumentId = "document_id"
-  val WordNameWordCount = "word_count"
   val DocumentCount = "document_count"
   val DocumentNameWordNameWordCount = "document_word_count"
 
+  val WordNameWordCount = "word_count"
+  val WordName = "word_name"
```

Comment:
I like that idea but, wouldn't be correct to do the same with all the Schema files we have to keep consistency? Or it's just because in essence Name and ID seem really similar?